jjb/jjb-templates/jq-core-jobs.yaml

```diff
@@ -0,0 +1,84 @@
+---
+- job_boiler_plate: &job_boiler_plate
+    name: job-boiler-plate
+
+    project-type: freestyle
+    node: "{build-node}"
+
+    ######################
+    # Default parameters #
+    ######################
+
+    branch: main
+    submodule-recursive: true
+
+    #####################
+    # Job Configuration #
+    #####################
+
+    properties:
+      - lf-infra-properties:
+        project: "{project}"
+        build-days-to-keep: "{build-days-to-keep}"
+
+    wrappers:
+      - lf-infra-wrappers:
+        build-timeout: "{build-timeout}"
+        jenkins-ssh-credential: "{jenkins-ssh-credential}"
+
+    publishers:
+      - lf-infra-publish
+
+- verify_boiler_plate: &verify_boiler_plate
+    name: verify_boiler_plate
+
+    concurrent: true
+
+    scm:
+      - lf-infra-github-scm:
+          url: '{git-clone-url}{github-org}/{project}'
+          refspec: '+refs/pull/*:refs/remotes/origin/pr/*'
+          branch: '$sha1'
+          submodule-recursive: '{submodule-recursive}'
+          choosing-strategy: default
+          jenkins-ssh-credential: '{jenkins-ssh-credential}'
+          submodule-disable: false
+          submodule-timeout: 10
+
+    triggers:
+      - github-pull-request:
+          trigger-phrase: '^recheck$'
```

Comment:
What does this mean?

Reply:
This allows the build to retrigger with the comment recheck